### ISSUE TITLE
Header menu button should have aria-expanded instead of aria-label

### ISSUE THREAD

**Issue by KaiSpencer** [NHSDigital/nhsuk-react-components#142]
aria-label="Open menu" -> aria-expanded="false" on the header menu button

---

See html in https://service-manual.nhs.uk/design-system/components/header

**Comment by KaiSpencer** [comment]
Closing as #167 to release the v5 upgrade branch is complete.
